repository/RediStick-Core.package/RsRedisEndpoint.class/instance/unifiedCommand.withDictionary..st

```diff
@@ -0,0 +1,3 @@
+private
+unifiedCommand: anArray withDictionary: aDictionary
```

Comment:
I think this method had no caller, maybe this should be included in other PR.

Reply:
fix 573893d